dashboard/utils.py

```diff
@@ -219,7 +229,7 @@ def has_paid_final_grade(self, edx_course_key):
         Returns:
             bool: whether a frozen final grade exists
```

Comment:
not related to this PR. this should be something like: "whether or not a user has a final grade and has paid"